prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -434,8 +434,19 @@ public async Task<IEnumerable<HpdidLookup>> HpdidLookupAsync(IEnumerable<string>
             return await _context.Enrollees
                 .Include(e => e.AccessTerms)
                 .Where(e => hpdids.Contains(e.HPDID))
+                .Where(e => e.CurrentStatus.StatusCode != (int)StatusType.Declined)
                 .Select(e => HpdidLookup.FromEnrollee(e))
                 .ToListAsync();
         }
+
+        public async Task<Enrollee> GetEnrolleeForUserIdNotDeclinedAsync(Guid userId)
```

Comment:
I think I would rather see an optional parameter on getenrolleeforuserid with excludeDeclined = false or something like that.